dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -308,12 +293,18 @@ def main(args):
         t_p.append(throughput)
         print_key_value(key=f"{m}", value=f"{format_bytes(throughput)}/s")
         t_runs[idx] = float(format_bytes(throughput).split(" ")[0])
-    t_p = numpy.asarray(t_p)
-    times = numpy.asarray(times)
+    t_p = np.asarray(t_p)
+    times = np.asarray(times)
+    bandwidths_all = np.asarray(bandwidths_all)
     print_separator(separator="=")
     print_key_value(
-        key="Throughput",
-        value=f"{format_bytes(t_p.mean())} +/- {format_bytes(t_p.std()) }",
+        key="Throughput (Total)",
+        value=f"{format_bytes(t_p.mean())} +/- {format_bytes(t_p.std())}",
+    )
+    print_key_value(
+        key="Throughput (Comms only)",
+        value=f"{format_bytes(bandwidths_all.mean())} +/- "
+        f"{format_bytes(bandwidths_all.std())}",
```

Comment:
Doesn't seem to be the case, although it's possible this is only for normal message passing (e.g., heartbeats):

```
$ DASK_DISTRIBUTED__COMM__COMPRESSION=None OPENBLAS_NUM_THREADS=1 python dask_cuda/benchmarks/local_cudf_merge.py -p ucx -d 0,1,2,3,4,5,6,7 -c 1_000_000 --runs 30 -b explicit-comms
...
Throughput                | 1.07 GiB/s +/- 209.63 MiB/s
Bandwidth                 | 103.27 MiB/s +/- 90.80 MiB/s
```

Reply:
So this must be background messaging since I don't think that the bandwidth can be lower than the throughput if we're actually measuring data movement of the computation.